java/service-feature-table-cache/README.md

```diff
@@ -1,18 +1,37 @@
-# Service Feature Table (Cache)
-Use a feature service with a service feature table in on-interaction-cache mode (which is the default mode for service feature tables). 
+# Service feature table (Cache)
 
-![Service FeatureTable Cache App](service-feature-table-cache.png)
+Display a feature layer from a service using the **on interaction cache** feature request mode.
+
+![Image of service feature table on interaction cache](service-feature-table-cache.png)
+
+## Use case
+
+`ServiceFeatureTable` supports three request modes, which define how features are requested from the service and stored in the local table. The feature request modes have different performance characteristics. Use **On interaction cache** in scenarios with large amounts of infrequently edited data. See [Table performance concepts](https://developers.arcgis.com/net/latest/wpf/guide/layers.htm#ESRI_SECTION1_40F10593308A4718971C9A8F5FB9EC7D) to learn more.
 
 ## How to use the sample
-Simply run the app.
+
+Run the sample and pan and zoom around the map. With each interaction, features will be requested and stored in a local cache. Each subsequent interaction will display features from the cache and only request new features from the service.
```

Comment:
Just a note, don't think you need to change anything: it seems like this sample automatically sets its viewpoint on the extent of the data so panning and zooming doesn't actually introduce new data

Reply:
good point: there's probably a sample enhancement idea in here, though the sample still does its job in showing the workflow and describing what is happening.